flink-cdc-common/src/main/java/org/apache/flink/cdc/common/utils/SchemaUtils.java

```diff
@@ -452,6 +455,43 @@ public static Schema inferWiderSchema(@Nullable Schema lSchema, Schema rSchema)
         return lSchema.copy(mergedColumns);
     }
 
+    public static void validateMetaSchemaCompatibility(LinkedHashSet<Schema> schemas) {
```

Comment:
I wonder if is this method really reusable enough to be a utility method?

Seems the argument type is quite specific (it must be a `LinkedHashSet` instead of a generic schemas' collection). Also, it didn't return a "Yes or No" boolean and let the caller to decide what to do, but throws an exception if it's incompatible.

Reply:
[but throws an exception if it's incompatible]
yeal,I think this kind of check is mandatory and cannot be ignored, and the exception is fatal, so I designed it to return void, but returning boolean seems to be good too. What do you think?

Reply:
My concern is if we're throwing an exception in the utility method, it might be less universal. But your implementation is more reasonable, because we can't return any reason message to indicate the incompatible reason by returning a boolean.

Let's just keep it as is.

Reply:
I have updated the code, could you please help review it again,thanks~